visualization/app/codeCharta/util/customConfigHelper.ts

```diff
@@ -267,12 +267,24 @@ export class CustomConfigHelper {
 
 		// TODO: remove this dirty timeout and set camera settings properly
 		// This timeout is a chance that CustomConfigs for a small map can be restored and applied completely (even the camera positions)
-		setTimeout(() => {
-			threeCameraService.setPosition()
-			threeOrbitControlsService.setControlTarget()
+		if (customConfig.camera) {
+			setTimeout(() => {
+				threeOrbitControlsService.setControlTarget(customConfig.camera.cameraTarget)
+				threeCameraService.setPosition(customConfig.camera.camera)
+			}, 100)
+		}
+	}
 
-			store.dispatch(setCamera(customConfig.stateSettings.appSettings.camera))
-			store.dispatch(setCameraTarget(customConfig.stateSettings.appSettings.cameraTarget))
-		}, 100)
+	// TODO [2023-01-01] remove support
+	private static transformLegacyCustomConfig(customConfig: any) {
```

Comment:
I would rename this method to express that the camera settings are transformed.

Reply:
done with next push